### ISSUE TITLE
Getting failed to parse source map from warnings on the several of the files

### ISSUE THREAD

**Issue by dvselva** [contentful/rich-text#326]
i am using conentful in react and i am getting the following warning when i run on dev mode.  

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\blocks.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\blocks.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\emptyDocument.ts' file: Error: ENOENT: no such file 
or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\emptyDocument.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\helpers.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\helpers.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\index.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\index.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\inlines.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\inlines.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\marks.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\marks.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\nodeTypes.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\nodeTypes.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\schemaConstraints.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\schemaConstraints.ts'

Failed to parse source map from 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\types.ts' file: Error: ENOENT: no such file or directory, open 'C:\Selva\Bootstrap\reactapp\myapp\node_modules\@contentful\rich-text-types\src\types.ts'

**Comment by mendrik** [comment]
downgrading to 14.1.2 fixed it for me

**Comment by dvselva** [comment]
yes. i tried 15.7.0 and it works and does not have these warnings

**Comment by amarcin** [comment]
How do you downgrade?

**Comment by tasmith039** [comment]
This is indeed quite annoying. would be good to have fixed

**Comment by VamOSGS** [comment]
Facing same issue with react-scripts v5 (project without typescript).

**Comment by Svarozic** [comment]
+1

**Comment by danielbeutner** [comment]
Seems the source maps reference to `../src/` but it is not packaged in node_modules.

Adding them could be solve this?
